cmd/githubgen/main.go

```diff
@@ -127,11 +129,22 @@ func loadMetadata(filePath string) (metadata, error) {
 	return md, nil
 }
 
-func run(folder string) error {
+func run(folder string, allowlistFilePath string) error {
+	members, err := getGithubMembers()
+	if err != nil {
+		return err
+	}
+	allowlistData, err := os.ReadFile(allowlistFilePath)
+	if err != nil {
+		return err
+	}
+	allowlist := strings.Split(string(allowlistData), "\n")
+
 	components := map[string]metadata{}
 	foldersList := []string{}
 	maxLength := 0
-	err := filepath.Walk(folder, func(path string, info fs.FileInfo, err error) error {
+	allCodeowners := make(map[string]struct{}, 24)
```

Comment:
Any reason for the magic number here?

Reply:
I picked something. Should I just not try to set a value?

Reply:
OK removed :)